subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/BuildScriptCompileAvoidanceIntegrationTest.kt

```diff
@@ -892,7 +892,10 @@ class BuildOperationsAssertions(buildOperationsFixture: BuildOperationsFixture,
     val bodyCompileOperations = buildOperationsFixture.all(Pattern.compile("Compile script build.gradle.kts \\(BODY\\)"))
 
     private
-    val compileAvoidanceWarnings = output.lines().filter { it.startsWith("Cannot use Kotlin build script compile avoidance with") }
+    val compileAvoidanceWarnings = output.lines()
+        .filter { it.startsWith("Cannot use Kotlin build script compile avoidance with") /*&& !it.contains("/distributions-basics/")*/ }
```

Comment:
Why leave the commented-out code in there?

Reply:
Unintentional.